jdi-light-vuetify-tests/src/test/java/io/github/epam/vuetify/tests/complex/TextFieldsTests.java

```diff
@@ -217,38 +228,46 @@ public void progressTextFieldTest() {
 
     @Test
     public void customValidationTextFieldTest() {
-        customValidationTextField.get(1).focus();
-        customValidationTextField.get(2).focus();
-        customValidationTextField.get(1).message().has().text("This field is required");
+        TextField firstCustomValidationTextField = customValidationTextField.get(1);
+        TextField secondCustomValidationTextField = customValidationTextField.get(2);
+        TextField thirdCustomValidationTextField = customValidationTextField.get(3);
+
+        firstCustomValidationTextField.focus();
+        secondCustomValidationTextField.focus();
+        firstCustomValidationTextField.message().has().text("This field is required");
 
-        customValidationTextField.get(2).sendKeys("My Keys");
-        customValidationTextField.get(1).message().has().text("Hey! I'm required");
+        secondCustomValidationTextField.sendKeys("My Keys");
+        firstCustomValidationTextField.message().has().text("Hey! I'm required");
 
-        customValidationTextField.get(2).clear();
-        customValidationTextField.get(2).message().has().text("This field is required");
+        secondCustomValidationTextField.clear();
+        secondCustomValidationTextField.message().has().text("This field is required");
 
-        customValidationTextField.get(2).sendKeys("123456789012345678901234567890");
-        customValidationTextField.get(2).message().has().text("Address must be less than 25 characters");
+        secondCustomValidationTextField.sendKeys("123456789012345678901234567890");
+        secondCustomValidationTextField.message().has().text("Address must be less than 25 characters");
 
-        customValidationTextField.get(3).sendKeys("My Keys");
-        customValidationTextField.get(3).clear();
-        customValidationTextField.get(3).message().has().text("This field is required");
+        thirdCustomValidationTextField.sendKeys("My Keys");
+        thirdCustomValidationTextField.clear();
+        thirdCustomValidationTextField.message().has().text("This field is required");
     }
 
     @Test
     public void fullWidthWithCounterTextFieldTest() {
         String FULL_WIDTH_CLASS = "v-text-field--full-width";
         String SINGLE_LINE_CLASS = "v-text-field--single-line";
```

Comment:
Change names, no caps.

Reply:
please follow naming conventions
if it's a common variable - use camelCase, if it's a constant - use 'final' and upper case

Reply:
Done